settings/installer/packstack/network/variant/_neutron_ml2-vlan.yml

```diff
@@ -0,0 +1,27 @@
+---
+plugin: ml2
+name: vlan
+internal:
+    label: 'physnet2'
+    vlan:
+        range:
+            start: 290
+            end: 340
+        bridge: "br-{{ !lookup installer.network.internal.label }}"
+
+config:
+    # 290-340 is the private vlan range assign to automation in qa.lab.tlv
```

Comment:
Please remove this notice (you can post post it somewhere else in our private repo)